docs/plugins/verifier/notation.md

```diff
@@ -260,3 +260,19 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
 ```
+
+## Certificate Revocation Check (CRL)
+
+Ratify supports CRL based on notation-core-go v1.2.0-rc1 and notation-go v1.3.0-rc1
```

Comment:
```suggestion
Ratify supports validating signing identity (certificate and certificate chain) revocation status using [certificate revocation evaluation](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#certificate-revocation-evaluation) as per revocation setting in Notation trust policy. The CRL implementation uses Notation libraries and follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#crls). 
```

Reply:
I think the versions of notation libraries are just implementation details which are not necessary for end users. We just need to tell what is Ratify CRL works and how it works in the doc.

Reply:
Thanks!